bin/openstack-install

```diff
@@ -110,8 +114,10 @@ def populate_config(opts):
         _cfg = yaml.load(utils.slurp(presaved_config))
     else:
         _cfg = {}
-    # Update cfg dict with command line opts
-    _cfg.update(vars(opts))
+    # Update cfg dict with command line opts unless
+    # we're in headless mode
```

Comment:
this is a little unexpected. so if you use --headless then any other cmdline opts are ignored?